src/sidecar.py

```diff
@@ -87,17 +91,17 @@ def main():
             v_minor = re.sub(r'\D', '', version.minor)
 
             if len(v_major) and len(v_minor) and (int(v_major) > 1 or (int(v_major) == 1 and int(v_minor) >= 19)):
-                print(f"{timestamp()} Ignore already processed resource version will be enabled.")
+                logger.debug("Ignore already processed resource version will be enabled.")
```

Comment:
Hi !
Perhaps you may log these ones in 'info' level and 'warning' for the api compatibility issue instead of debug.

Thanks.
Regards.

Reply:
Good catch ! Thanks